configs/components/puppet-ca-bundle.json

```diff
@@ -1,4 +1,4 @@
 {
-  "url": "git://github.com/puppetlabs/puppet-ca-bundle.git",
+  "url": "git@github.com:puppetlabs/puppet-ca-bundle.git",
```

Comment:
Seems like we should fix those URLs in stable (in a separate PR) and merge that up to master, so we don't get different behavior across branches. /cc @melissa @underscorgan 

:+1: to merge this PR as-is